components/Forms/ProductForm.js

```diff
@@ -0,0 +1,219 @@
+import { defaultImageURL } from "@/public/images/defaultImageURL";
+import Link from "next/link";
+import { uid } from "uid";
+import {
+  StyledForm,
+  StyledTitleInput,
+  StyledLabel,
+  StyledInput,
+  StyledTextArea,
+} from "@/components/Forms";
+import {
+  StyledSubmitButton,
+  StyledCancelButton,
+  StyledButtonContainer,
+  StyledIconButton,
+} from "@/components/Buttons";
+import Icon from "@/components/Icons";
+import styled from "styled-components";
+import { useState } from "react";
+import Loading from "../Loading";
```

Comment:
Super minor thing, but is there any reason why you use @ for imports above and then ../ here?